packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -1014,6 +1014,18 @@ export const instructionTypeDefs = /* GraphQL */ `
         withheldAmount: String
     }
 
+    """
+    SplToken-2022: InitializeTokenGroup instruction
+    """
+    type SplTokenInitializeTokenGroup implements TransactionInstruction {
+        programId: Address
+        group: Account
+        maxSize: BigInt
+        mint: Account
+        mintAuthority: Account
+        updateAuthority: Account
+    }
+
```

Comment:
So, this instruction set is actually an [interface](https://github.com/solana-labs/solana-program-library/blob/master/token-group/interface/src/instruction.rs), so is SPL Token Metadata, so let's actually define this separately from Token-2022.

```suggestion
    """
    Spl Token Group: InitializeGroup instruction
    """
    type SplTokenGroupInitializeGroup implements TransactionInstruction {
        programId: Address
        group: Account
        maxSize: BigInt
        mint: Account
        mintAuthority: Account
        updateAuthority: Account
    }
```